cfme/tests/cloud/test_providers.py

```diff
@@ -1409,3 +1409,58 @@ def test_add_ec2_provider_with_sts_assume_role(appliance, ec2_provider_with_sts_
     """
     ec2_provider_with_sts_creds.create()
     ec2_provider_with_sts_creds.validate()
+
+
+@pytest.mark.ignore_stream("5.10")  # BZ 1710623 was not merged into 5.10
```

Comment:
Is this test case providing coverage for BZ 1710623? If so can you add it as automates coverage?

Reply:
It is not. In that BZ was changed formatting of regions so I can compare regions across CFME.

Reply:
Well, I was thinking about it a little more and I added automate coverage for it, because it tests this issue as side-effect.